mindsdb/integrations/handlers/anthropic_handler/README.md

```diff
@@ -0,0 +1,24 @@
+# Anthropic Handler
+Anthropic ML handler integrates Anthropic LLMs with MindsDB. You can train your existing text data with Anthropic LLM
+
+# Anthropic
+Anthropic is a AI assitant framework for your usecases. In this handler, we use the Anthropic package, which is available in Python. More information about this Python client can be found (here)[https://github.com/anthropics/anthropic-sdk-python].
```

Comment:
Please correct this paragraph, there are a number of imprecisions here:

* Anthropic is an AI safety and research company. Claude is their assistant model.
* The package used is the Anthropic Python SDK.
* A bunch of typos, like "assitant" or "usecases". I recommend using something like Grammarly to detect these for an easy fix.

Reply:
I would additionally put a disclaimer in bold for users to first ensure they've read the evaluation terms that Anthropic defines [here](https://docs.anthropic.com/claude/reference/getting-started-with-the-api#evaluation--going-live-with-the-api).

Reply:
Thanks for the review, I have made the necessary changes